package.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "ecommerce",
+  "private": true,
+  "version": "1.0.0",
+  "description": "",
+  "main": "main.ts",
+  "type": "module",
+  "scripts": {
+    "test": "vitest",
+    "dev": "vite",
+    "lint": "eslint ./src",
+    "lint:fix": "eslint --fix ./src",
+    "typecheck": "tsc",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "format": "prettier --write 'src/**/*.{ts,tsx,css,scss}'",
```

We don't need the tsx extension here since we aren't using React components
```suggestion
    "format": "prettier --write 'src/**/*.{ts,css,scss}'",
```